.github/workflows/melange-test-pipelines.yaml

```diff
@@ -74,6 +74,27 @@ jobs:
           go-version-file: './go.mod'
           check-latest: true
 
+      - name: Download kernel for VMs
+        run: |
+          KERNEL_PKG="$(curl -sL https://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz | tar -Oxz APKINDEX | awk -F':' '$1 == "P" {printf "%s-", $2} $1 == "V" {printf "%s.apk\n", $2}' | grep "linux-virt" | grep -v dev)"
+          curl -LSo linux-virt.apk "https://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/$KERNEL_PKG"
```

Comment:
does it make sense to pin to a version here? and verify the checksum?

Reply:
Good point. I think that would be easier to take the latest, to reduce maintenance since we're not really depending on specific kernel features for these tests. Also, latest provided by linux-virt is an LTS (ref https://wiki.alpinelinux.org/wiki/Kernels)